### PR TITLE
API-95: balances api

### PR DESCRIPTION
**Pull request by laher**
This PR is for Gift Cards - BETA

### CODE REVIEW

File: api.json [hunk 1]
```diff
@@ -1359,6 +1359,276 @@
         }
       }
     },
+    "/gift_cards": {
```
Comment (laher) [inline api.json]: let's revert the changes to this file - to just use api-beta.json

**Review by laher — COMMENTED** [review]
Looks great now. Woop woop!

**Review by quantumchaos451 — APPROVED** [review]
Looks good!
